samtranslator/third_party/py27hash/hash.py

```diff
@@ -78,8 +78,7 @@ def thash(value):  # type: ignore[no-untyped-def]
         for y in value:
             length -= 1
 
-            y = Hash.hash(y)
-            x = (x ^ y) * mult
+            x = (x ^ Hash.hash(y)) * mult
```

Comment:
What did it complain about? Reusing `y`? 2 assignments was clearer to me.

Reply:
See description